src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.sscs.idam.IdamService;
+import uk.gov.hmcts.reform.sscs.idam.IdamTokens;
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingDeleteRequestPayload;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingResponse;
+
+import javax.servlet.http.HttpServletResponse;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+
+@ExtendWith(MockitoExtension.class)
+public class HearingsServiceTest {
+
+    @Mock
+    private HmcHearingApi hmcHearingApi;
+
+    @Mock
+    private IdamService idamService;
+
+    private HearingsService hearingsService;
```

Comment:
If you annotate this with @InjectMocks you won't need the openMocks below

Reply:
Done